Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mhart/alpine-node:base
+
+ENV PATH /root/.yarn/bin:$PATH
+
+RUN apk update \
+  && apk add curl bash binutils tar git \
+  && rm -rf /var/cache/apk/* \
+  && /bin/bash \
+  && touch ~/.bashrc \
+  && curl -o- -L https://yarnpkg.com/install.sh | bash \
+  && git clone https://github.com/rage8885/OpenSC2K \
```

Comment:
With the Dockerfile being in the repository, git cloning is not needed. We can just `ADD . /home/node/app`